RoslynPluginGenerator/DiagnosticAssemblyScanner.cs

```diff
@@ -31,15 +30,14 @@ public DiagnosticAssemblyScanner(ILogger logger, params string[] additionalSearc
         }
 
         /// <summary>
-        /// Loads all assemblies in the given directory and instantiates Roslyn diagnostic objects - i.e. existing types deriving from
+        /// Loads all of the given assemblies and instantiates Roslyn diagnostic objects - i.e. existing types deriving from
         /// <see cref="DiagnosticAnalyzer"/>
         /// </summary>
         /// <returns>Enumerable with instances of DiagnosticAnalyzer from discovered assemblies</returns>
-        public IEnumerable<DiagnosticAnalyzer> InstantiateDiagnostics(string directoryPath, string language)
+        public IEnumerable<DiagnosticAnalyzer> InstantiateDiagnostics(string language, params string[] files)
```

Comment:
Previously we searching a directory for all assemblies. Now we use the list of assemblies returned from extracting the NuGet package to a known location.